packages/manager-react-components/src/components/templates/onboarding/onboarding.component.tsx

```diff
@@ -14,17 +14,18 @@ import placeholderSrc from '../../../../public/assets/placeholder.png';
 type OnboardingLayoutButtonProps = {
   orderButtonLabel?: string;
   orderHref?: string;
-  moreInfoHref?: string;
-  moreInfoIcon?: ODS_ICON_NAME;
-  moreInfoButtonLabel?: string;
   onOrderButtonClick?: () => void;
-  onmoreInfoButtonClick?: () => void;
   isActionDisabled?: boolean;
   orderIam?: {
     urn: string;
     iamActions: string[];
     displayTooltip?: boolean;
   };
+  moreInfoHref?: string;
+  moreInfoButtonIcon?: ODS_ICON_NAME;
+  moreInfoButtonLabel?: string;
+  onmoreInfoButtonClick?: () => void;
```

Comment:
```suggestion
  onMoreInfoButtonClick?: () => void;
```

Reply:
Yes I put both props and deprecated this one to avoid breaking changes